beaconChain/helpers/stateTransitionHelpers.ts

```diff
@@ -402,7 +403,8 @@ export function getBeaconProposerIndex(state: BeaconState, slot: int): int {
  * @returns {boolean}
  */
 export function isPowerOfTwo(value: int): boolean {
-  return value === 0 ? false : 2 ** (Math.log2(value)) === value;
+  assert(value >= 0);
```

Comment:
Hey thanks for the contribution!

Two things:
1) We are trying to be a pure JS (no nodejs) implementation, so we can't use assert (I'll clarify this in the readme further)
2) We should return false instead of asserting in this instance.

Reply:
You can also use `throw new Error("[isPowerOfTwo()]Number smaller <= 0")`

Reply:
I think it's better to throw an error, because something has gone terribly wrong if we are passing in a negative value ( and returning false would hide that).